plugins/system76-launch/fu-system76-launch-device.h

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright (C) 2019 Richard Hughes <richard@hughsie.com>
```

Comment:
ohh you can remove my copyright here if you like too -- i wasn't really involved at all :)

Reply:
This was copied from thelio-io, so I think that should stay